modules/client/src/client-common/utils.ts

```diff
@@ -78,7 +78,7 @@ export function computeProposalStatusFilter(
 }
 
 export function isProposalId(propoosalId: string): boolean {
-  const regex = new RegExp(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,}$/i);
+  const regex = new RegExp(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{64}$/i);
```

Comment:
If we keep a dynamic nonce hex size, this could remain

Reply:
Improved the regex to be of max 32 bytes length